### DISCUSSION TITLE
Commenting out Anki cards with html tags

### DISCUSSION THREAD

**Discussion by base-I** [ObsidianToAnki/Obsidian_to_Anki#211] [Ideas]
Hey,

would it be possible to comment out tags by using html tags? Atm this is not possible as the PlugIn thinks html comment tags are meant to ignore the note ID and thus completely ignores them? (See attached picture)

I get that your use case is inline/embedded anki cards, but I might want to use obsidian publish at some point and it would be very useful to not have cards inline then :). Maybe this could be done with a little adjustment of the note ID regex? 

I tried to do this myself but I could not do it :)

Cheers!
<img width="439" alt="Bildschirmfoto 2021-02-05 um 10 36 23" src="https://user-images.githubusercontent.com/75250395/107016238-fef61a80-679d-11eb-8822-121439924f6d.png">

**Comment by base-I** [comment]
This can now be achieved in v0.10.12 by wrapping the "card" into

%%
_content here_
%%

**Reply by rob-hamill** [reply]
@base-I  This may be useful for me, but could you please provide a few examples so I can fully understand? I am going to post a new post in discussion with particular problem I am facing. Thanks!

**Reply by base-I** [reply]
Content is German but I only used standard "syntax" so it should be fine as an example. These will only show up in edit-mode but not Preview mode :) Hope this helps!


<img width="473" alt="Bildschirmfoto 2021-02-28 um 09 35 29" src="https://user-images.githubusercontent.com/75250395/109412448-5ad04f80-79a8-11eb-8c7f-015eaa327c7f.png">

**Reply by rob-hamill** [reply]
Thanks! This is helpful
